pkg/apis/eksctl.io/v1alpha5/types.go

```diff
@@ -489,4 +491,21 @@ type (
 		// +optional
 		PublicKeyName *string `json:"publicKeyName,omitempty"`
 	}
+
+	// MixedInstances holds the configuration for spot instances
+	// TODO: validation of base capcity : make required
+	// TODO allocation strategy?
+
+	MixedInstances struct {
```

Comment:
```suggestion
	NodeGroupInstancesDistribution struct {
```
So it's in line with e.g. `NodeGroupSSH`.